sample/java/src/main/java/com/datadog/android/sample/MainActivity.java

```diff
@@ -187,5 +189,27 @@ private void addSpanInScope(String opName, Runnable execute) {
             span.finish();
         }
     }
+
+    private static class FragmentsFactory {
```

Comment:
Ain't this overkill ? 
Couldn't you use a generic bundle with, say the menu item id as argument? 
You could have kept the code above and then simply added the bundle arguments before starting the FragmentTransaction ?

Reply:
Yeah...forgot to remove that....wanted to do a factory for some reason, then dropped the reason and forgot to remove the factory.